exams/factories.py

```diff
@@ -20,3 +32,27 @@ class ExamProfileFactory(DjangoModelFactory):
 
     class Meta:  # pylint: disable=missing-docstring
         model = ExamProfile
+
+
+class ExamAuthorizationFactory(DjangoModelFactory):
+    """
+    Factory for ExamAuthorization
+    """
+    user = SubFactory(UserFactory)
+    course = SubFactory(CourseFactory)
+
+    operation = FuzzyChoice(
+        [value[0] for value in ExamAuthorization.OPERATION_CHOICES]
+    )
+    status = FuzzyChoice(
+        [value[0] for value in ExamAuthorization.STATUS_CHOICES]
+    )
+    date_first_eligible = factory.LazyAttribute(
+        lambda x: FAKE.date_time_this_year(before_now=False, after_now=True, tzinfo=pytz.utc).date()
```

Comment:
We have both usages in our code, are we standardizing towards one over the other? https://github.com/mitodl/micromasters/search?utf8=%E2%9C%93&q=pytz.utc